Automattic/jetpack
# Build: Use `build-concurrently` scripts in Travis

### Changes proposed in this Pull Request:

* Add `build-concurrently` build scripts
* Use `build-concurrently` scripts in CI

#### Jetpack product discussion


n/a
#### Does this pull request change what data or activity we track or use?


n/a
#### Testing instructions:






* Make sure Travis is still green
* Compare total build times with other builds for `Build dashboard & extensions` and E2E tests

#### Proposed changelog entry for your changes:

* n/a
